src/xtgeo/common/sys.py

```diff
@@ -421,7 +421,7 @@ def get_cfhandle(self) -> int:
 
         if _cfhandle already exists, then _cfhandlecount is increased with 1
 
-        Returns:
+        Returns
             int indicating the file handle number.
```

Comment:
Why remove ":"? They shall be there accdrong to https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html